### ISSUE TITLE
Attaching and Programing the Forearm and Claw Assembly

### ISSUE THREAD

**Issue by Geckoing** [sta-titansrobotics/27315-IntoTheDeep#18]
Now that sprint 1 is over the next steps are to finish building the main robot and code it to be competition ready.

**Comment by Geckoing** [comment]
Tuesday:

I made it to the final step of the Forearm and Claw Assembly and had to wait for Anand and Mateo to cut the 400mm into the 2 pieces of 175mm and 225mm so I could build my final steps and they could as well. A good amount of work was done and it can be finished tomorrow.

**Comment by Geckoing** [comment]
Wednesday:

I pretty much finished the Forearm and Claw Assembly however I encountered an issue where the specified chain link length in the instructions was too short compared to what I needed. This created to much tension on the chain and made it almost impossible to attach. After completing one attachment it was clear that I would need to take creative liberties in order to finish the assembly. Sadly figuring things out took almost the entire period and I ran out of time which is disappointing because I am now 2 days behind in technicality.

**Comment by Geckoing** [comment]
Thursday:

I used the 1st part of the period before lunch to completely finish the Forearm and Claw Assembly. It took a little longer as Ambrose tried fixing some elements that didn't need fixing but that's beside the point. Now that the assembly is completed worked with the rest of the team to attach the different assembly's to the main drivetrain. Finally I worked on these github updates as well as helping out wherever needed.

**Comment by Geckoing** [comment]
Friday:

Today was mostly spent discussing how we will attach the intake to the main assembly given that we hope to have some sort of extension method to reach the high basket. The main idea that came out of the discussion was using gear belts and multiple other extrusions to create a folding compact design that can than extend using motors and reach wherever necessary. We then drew up some sketches on the whiteboard with Mr. Fabroa's help before then testing different places for the gear belts on the actual robot.

**Comment by Geckoing** [comment]
22/10/24 Tuesday:

Today's period was spent modifying the robot to help see how the new extended intake would work. We figured out that we will need another hex shaft about 180mm long but as we don't have extra we will be cutting one already on the robot to fit our new requirements while still allowing the correct size rod to hold its original use. We also spent a large chunk of the period discussing how the newly confirmed tournament dates will fit for all of us.

**Comment by Geckoing** [comment]
23/10/24 Wednesday:

We came to class today to find the fourbar and intake completely disassembled. This is because Ambrose decided to completely disassemble it without asking anyone. So because of that we spent most of the period re-doing what Ambrose took apart. Personally I helped improve the intake by adding a rear wall to make sure any samples don't go flying out the back of the intake as well as assisting with various other parts.

![image](https://github.com/user-![image](https://github.com/user-attachments/assets/5cd807a2-c839-4878-b49f-779813761503)attachments/assets/e01ccbdd-cd22-4d75-b986-ec377852ba03)

**Comment by Geckoing** [comment]
24-25/10/24 Thursday and Friday:

Nothing major happened as I just helped out with various tasks like disassembling our old robot. I also assisted with some assembly as well as pre-loading brackets and various other pieces used in the construction of the four bar assembly. I am currently waiting on the completion of the revisions to our robot so I can evaluate where we are from a driver perspective. The main goal moving forward is to see how we can achieve a high level ascent.
